src/dsc.h

```diff
@@ -420,6 +420,27 @@ class IdList {
         std::inplace_merge(begin(), end() - 1, end(), Compare());
     }
 
+    void MergeInto(IdList *dest) {
+        dest->ReserveMore(n);
+
+        const auto oldEnd = dest->elem + dest->n;
+        auto outIter      = oldEnd;
+
+        for(auto &elt : *this) {
+            // Copy-construct at the end of the list.
+            new(outIter) T(elt);
+            ++outIter;
+        }
+
+        /// @todo Look to see if we already have something with the same handle value.
```

Comment:
I'm not really comfortable merging this without an assert here.

Reply:
yeah I just realized this messes up a bunch of things if you actually try using a lathe. I'm dropping it, sorry.